### ISSUE TITLE
Fill out CfA Follow-Up Form

### ISSUE THREAD

**Issue by darrenrichmond** [codefordenver/ndoch2016#65]
https://cfa.typeform.com/to/cW7F3I

Questions I need some help on:
- Number of participants: I put 45 for now, but we could get an accurate count from the sign in sheets
- Length in Hours: I put 10 for now - do you know what time they ended?
- Please share the link(s) to the list of prototypes/projects/activities you worked on during National Day of Civic Hacking
- Please share any write-ups or press URLs from your event (blogs, local press coverage, etc) - Did we do any of that?
- Tell us about one project you're really happy about
- Name one project your group worked on that best represents your community and is something you would consider a success
  - short description
  - Link
  - Photo of team
- Please share one thing we can improve for our next days of action

**Comment by Mira321** [comment]
Not sure how to help update this.  Looks like you have to start all over when I click on the link.  Is there a link to the saved file of what's been done?

**Comment by Mira321** [comment]
The link for photo's is  http://www.meetup.com/CodeForDenver/photos/27027143/

**Comment by Mira321** [comment]
Willy has the sign-in sheets for the exact # but I think it was around 30 - I would keep the number you entered in though - there may have been some who did not sign in.

**Comment by Mira321** [comment]
I would mention our speaker from Metro Caring - Luis Linares - who came to share housing issues from their perspective - and spoke for about 45 min.

**Comment by Mira321** [comment]
Particular project we are proud of is the app to track available beds for those seeking shelter for the night.  See FlowDoc post on June 5th by Rain - you can probably copy the image.  Not sure where this is going but it was an exciting project with a lot of enthusiasm generated on NDoCH.

**Comment by Mira321** [comment]
Also - Greg's project - not sure what he is calling it, I think it is Open Source Circular Economy.  

The success is that these two projects we initiated and have been added to CFD project board.

**Comment by Mira321** [comment]
I would leave the time to 10 hours - all in all when consider set up and clean up - that's about right.

**Comment by Mira321** [comment]
You might want to review the Debrief doc for suggestions that were mentioned by participants, but that may pertain more to our local group, not the National Group.  Willy, Karen or Dave might have better ideas of what the Natl group could do better with.

**Comment by wdoug** [comment]
**Number of participants** 
_Do I have the sign in sheet? I guess I will have to look for where it ended up. I'd agree with leaving it at 45_

**Please share the link(s) to the list of prototypes...**
_I'm not sure what they are asking for here_

**Please share any write-ups or press URLs**
_We haven't done that yet, although there was mention that it would be a good idea_

**Tell us about one project you're really happy about**
_Also, the data visualization project that Jake and Henry worked on is also a new CfD project. So in all, we got three new projects in exploration from the event_

I don't think I have anything for the other questions.

**Comment by Mira321** [comment]
I don't think we need to do any write ups at this point.  Willy - I think I gave you or maybe David, the 2 sign-in sheets.  At least I don't have them.  The pic from Rain's post could serve for a "prototype" I would think - maybe? If not we have listed the 3 projects that were worked on, so that should be sufficient.
